### ISSUE TITLE
not possible to initialize encrypted conversation

### ISSUE THREAD

**Issue by rynnon** [ReneVolution/profanity-omemo-plugin#36]
OMEMO encryption actually works for me. The issue is that I can't send encrypted messages until the person I want to talk with sent me an encrypted message, even though I already had encrypted conversations with them in the past.

Profanity command:

/omemo start contactX@some-jabber-server.org

// profanity output: 
OMEMO Session started.
###############################################
#                                             #
#                   CAUTION                   #
#    This plugin does not support any form    #
#             of Trust Management             #
#      All Devices are trusted *blindly*      #
#                                             #
###############################################

// I send a message:
Last message was sent unencrypted.
me: hello

//contact sends me a encrypted message message (they, which means me with a different account from my phone running conversations)
contactX@some-jabber-server.org: hey!
me: NOW this message is encrypted.

Is this actually a limitation of profanity/omemo plugin, or is there some other issue?

**Comment by licaon-kter** [comment]
Did you `/sub allow` ?

**Comment by rynnon** [comment]
yes. did it again for good measure, restarted profanity again. same as before.
